pack/td.json

```diff
@@ -489,6 +617,80 @@
         "uniqueness": false
     },
     {
+        "code": "13035",
+        "cost": 2,
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "faction_cost": 3,
+        "flavor": "\"Strictly speaking, it isn't illegal to use an interrogation simulator as the blueprint for ICE.\" - Mason Bellamy",
+        "illustrator": "Andreas Zafiratos",
```

Comment:
I'm not sure if this a typo at all, the card has the illustrator's surname as _**Zafeiratos**_, but a quick search on [deviant art](http://albino-z.deviantart.com/) points to the name you have inputed. 😕